README.md

```diff
@@ -4,13 +4,248 @@ NATS Streaming is an extremely performant, lightweight reliable streaming platfo
+	                             verification; weaker than specifying certificates.
+    -tls_client_key              Client key for the streaming server
+    -tls_client_cert             Client certificate for the streaming server
+    -tls_client_cacert           Client certificate CA for the streaming server
+
+Streaming Server Logging Options:
+    -SD, --stan_debug            Enable STAN debugging output
+    -SV, --stan_trace            Trace the raw STAN protocol
+    -SDV                         Debug and trace STAN
+    (See additional NATS logging options below)
+
+Embedded NATS Server Options:
+    -a, --addr <host>                Bind to host address (default: 0.0.0.0)
+    -p, --port <port>                Use port for clients (default: 4222)
+    -P, --pid <file>                 File to store PID
+    -m, --http_port <port>           Use port for http monitoring
+    -ms,--https_port <port>          Use port for https monitoring
+    -c, --config <file>              Configuration file
+
+Logging Options:
+    -l, --log <file>                 File to redirect log output
+    -T, --logtime                    Timestamp log entries (default: true)
+    -s, --syslog                     Enable syslog as log method
+    -r, --remote_syslog <addr>       Syslog server addr (udp://localhost:514)
+    -D, --debug                      Enable debugging output
+    -V, --trace                      Trace the raw protocol
+    -DV                              Debug and trace
+
+Authorization Options:
+        --user <user>                User required for connections
+        --pass <password>            Password required for connections
+        --auth <token>               Authorization token required for connections
+
+TLS Options:
+        --tls                        Enable TLS, do not verify clients (default: false)
+        --tlscert <file>             Server certificate file
+        --tlskey <file>              Private key for server certificate
+        --tlsverify                  Enable TLS, very client certificates
+        --tlscacert <file>           Client certificate CA for verification
+
+NATS Clustering Options:
+        --routes <rurl-1, rurl-2>    Routes to solicit and connect
+        --cluster <cluster-url>      Cluster URL for solicited routes
+
+Common Options:
+    -h, --help                       Show this message
+    -v, --version                    Show version
+        --help_tls                   TLS help.
+```
+
+## Securing NATS Streaming Server
+
+### Authorization
+
+If you use basic authorization, that is a single user defined through the configuration file or command line parameters, nothing else is required. The embedded NATS Server is started with those credentials and the NATS Streaming Server uses them.
+
+However, if you define multiple users, then you **must** specify the username and password (or token) corresponding to the NATS Streaming Server's user.
```

Comment:
May need to provide link here to the new docs on this, when they are done, since it is a new feature for NATS Server.